scripts/check_configs.ts

```diff
@@ -596,6 +675,38 @@ async function main() {
         outcomeMap[network] = false;
       }
 
+      // query all bridges
+      const bridgesMap: BridgeMap | undefined = sdk?.token?.bridges
+      let bridgesArr: string[] = []
+
+      if (bridgesMap && bridgesMap.ibc.length && bridgesMap.polynetwork.length) {
```

Comment:
1. Think you shouldn't check if there are both IBC Bridges AND Polynetwork Bridges cos what if there are only Polynetwork bridges but no IBC bridges.
2. Don't forget to check `bridgesMap.axelar.length` anyways
i.e.
```typescript
if (bridgesMap && (bridgesMap.ibc.length || bridgesMap.polynetwork.length || bridgesMap.axelar.length)) { ... }
```